src/components/ScheduleVisit/index.jsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable react/prop-types */
+import Button from 'components/shared/Button';
+import React, { forwardRef, useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import Slider from 'react-slick';
+import data from './data';
+import {
+  DateButton,
+  DateCard,
+  DateSliderContainer,
+  MainContainer,
+  SubmitDateSection,
+  Title,
+} from './schedule-visit.styles';
+
+/**
+ * Listing page schedule-visit section.
+ *
+ * @returns {JSX.Element}
+ */
+function ScheduleVisit() {
+  const [startDate, setStartDate] = useState(new Date());
+  const CustomInput = forwardRef(({ value, onClick }, ref) => (
+    <DateButton type="button" onClick={onClick} ref={ref}>
+      {value}
+    </DateButton>
+  ));
+
+  return (
+    <MainContainer>
+      <Title>Schedule Visit</Title>
+      <DatePicker
+        selected={startDate}
+        onChange={(date) => setStartDate(date)}
+        customInput={<CustomInput />}
+      />
+      <DateSliderContainer>
+        <Slider
+          arrows
+          dots={false}
+          infinite={false}
+          slidesToScroll={7}
+          slidesToShow={7}
+          responsive={[
```

Comment:
can we create a constant out side the component that has these values 
```suggestion
          responsive={responsive}
```